*** a/source/adapters/CMakeLists.txt	2025-07-25 05:56:47.844597496 +0200
--- b/source/adapters/CMakeLists.txt 	2025-07-25 16:29:19.571987092 +0200
***************
*** 26,35 ****
--- 26,44 ----

          set(ADAPTER_VERSION_SCRIPT ${CMAKE_CURRENT_BINARY_DIR}/${name}.map)

+         message(WARNING "test ${TARGET_LIBNAME} tset")
          configure_linker_file(
              ${CMAKE_CURRENT_SOURCE_DIR}/../adapter.map.in
              ${ADAPTER_VERSION_SCRIPT} @ONLY
          )
+
+         execute_process(
+             #COMMAND cat ${ADAPTER_VERSION_SCRIPT}
+             COMMAND ls ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_BINARY_DIR}/.. /build/source/build/source/adapters /build/source/build/source /build/source/build
+             OUTPUT_VARIABLE fungus
+         )
+         message(WARNING "amogus: ${fungus} sus?")
+
          target_link_options(${name} PRIVATE
              "-Wl,--version-script=${ADAPTER_VERSION_SCRIPT}"
          )
