diff --git a/cmake/helpers.cmake b/cmake/helpers.cmake
index de22fe75..517aff18 100644
--- a/cmake/helpers.cmake
+++ b/cmake/helpers.cmake
@@ -279,11 +279,13 @@ function(configure_linker_file input output)
     configure_file(${input} ${tmp} ${ARGN})
     # Strip guarded lines and capture stripped content from stdout
     execute_process(
-        COMMAND ${PYTHON_EXECUTABLE}
+        COMMAND ${Python3_EXECUTABLE}
             ${PROJECT_SOURCE_DIR}/scripts/strip-guarded-lines.py ${tmp}
             # List names of guarded blocks to include in the output file here
         OUTPUT_VARIABLE stripped
     )
+    message(WARNING "asdf ${stripped} fghj")
+    message(WARNING "asdf2 ${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR} ${tmp} fghj")
     # Write stripped output to file for use by the linker
     file(GENERATE OUTPUT ${output} CONTENT "${stripped}")
 endfunction()
